### ISSUE TITLE
FileNotFoundError: model-index.ym

### ISSUE THREAD

**Issue by Bezdarnost** [likyoo/open-cd#119]
FileNotFoundError: [Errno 2] No such file or directory: '../open-cd/model-index.yml'

**Comment by Aoldcat1997** [comment]
You give function a bad model path lead to  this problem

**Comment by Bezdarnost** [comment]
Possibly I faced with project building problem as mentioned in another question 

#116

**Comment by Aoldcat1997** [comment]
i alse faced the problem.
base ”opencd not in {'mmcls': 'mmcls', 'mmdet': 'mmdet', 'mmdet3d': 'mmdet3d', 'mmseg': 'mmsegmentation', 'mmaction': 'mmaction2', 'mmtrack': 'mmtrack', 'mmpose': 'mmpose', 'mmedit': 'mmedit', 'mmocr': 'mmocr', 'mmgen': 'mmgen', 'mmfewshot': 'mmfewshot', 'mmrazor': 'mmrazor', 'mmflow': 'mmflow', 'mmhuman3d': 'mmhuman3d', 'mmrotate': 'mmrotate', 'mmselfsup': 'mmselfsup', 'mmyolo': 'mmyolo', 'mmpretrain': 'mmpretrain', 'mmagic': 'mmagic'}!,please pass a valid scope.“，you can append the "opencd" in scope list,but you should find the list position ,it locate in  your 'env' libiary ,you need find it.

**Comment by Bezdarnost** [comment]
@likyoo please, help us, multiple independent people face with the same problem

**Comment by Aoldcat1997** [comment]
My root ：![image](https://github.com/user-attachments/assets/66c27e0b-ed17-41c7-8a7d-72fce189aec8)

**Comment by likyoo** [comment]
I can't reproduce the error, could you provide more details?

**Comment by likyoo** [comment]
pleaze make sure your `model` path is correct.

**Comment by zhanzf** [comment]
> pleaze make sure your `model` path is correct.

@likyoo you are right, when I correct the model and weight path, the problem solved!

Detail see #116
